src/game/objects/general/pickup.c

```diff
@@ -49,16 +49,44 @@ static const OBJECT_BOUNDS m_PickUpBoundsUW = {
     },
 };
 
+static void PickUp_AddGunToMesh(
+    const GAME_OBJECT_ID obj_num, ITEM_INFO *lara_item);
 static void PickUp_GetItem(
     int16_t item_num, ITEM_INFO *item, ITEM_INFO *lara_item);
 static void PickUp_GetAllAtLaraPos(ITEM_INFO *item, ITEM_INFO *lara_item);
 
+static void PickUp_AddGunToMesh(
+    const GAME_OBJECT_ID obj_num, ITEM_INFO *lara_item)
```

Comment:
`const ITEM_INFO *const lara_item`

Reply:
So do we use full `const` usage if it's a static function declaration?